app/boards/arm/puchi_ble/arduino_pro_micro_pins_flipped.dtsi

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Missed this the first time... Are you actually using this flipped file at all? Seems unnecessary.

Reply:
You are right, i deleted it.